### PR TITLE
docs: translate astro vs other docs to zh comparing-astro-vs-other-tools.md

### PR DESCRIPTION
**Pull request by ChelesteWang**
## Changes
docs: translate astro vs other docs to zh
comparing-astro-vs-other-tools.md

- What does this change?
- Be short and concise. Bullet points can help!
- Before/after screenshots can be helpful as well.

### CODE REVIEW

File: docs/src/pages/zh-CN/getting-started.md [hunk 1]
```diff
@@ -50,14 +61,24 @@ Astro 是一个现代的静态网站生成工具. 你可以从 [我们的主页]
 
 [@astrodotbuild](https://twitter.com/astrodotbuild) Twitter 账户是 Astro 团队的官方更新来源。
 
+<<<<<<< HEAD
```
Comment (matthewp) [inline docs/src/pages/zh-CN/getting-started.md:64]: I looks like there are quite a few merge conflicts in these changes, can you resolve these please?

Comment (ChelesteWang) [reply docs/src/pages/zh-CN/getting-started.md:64]: Of course, sorry for not noticing a conflict with the main branch

**Comment by ChelesteWang** [comment]
Gatsby was added to the translated document, which was not available at the time of translation

**Comment by FredKSchott** [comment]
LGTM!

**Pull request merged**